src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -568,6 +607,17 @@ abstract class IntegrationTestBase {
     )
     reviewScheduleHistoryRepository.saveAndFlush(reviewScheduleEntity)
   }
+
+  fun sendDomainEvent(
```

Comment:
Moved this method from the original `InboundEventListenerTest` so that we can use it in other event listener tests